src/common/lang.js

```diff
@@ -178,7 +178,8 @@ const lang = {
   NO: 'No',
   NOOFDEVICES: 'NO. Devices',
   NOTE: 'Note: Any changes made to rules will only apply to future events on the affected devices.',
-  NOTIFICATIONTYPE: 'Notification Type',
+  NOTIFICATION : 'Notifications',
+  NOTIFICATIONTYPE: 'NOTIFICATION Type',
```

Comment:
Should NOTIFICATION be all caps? Seems like something that should be done in the CSS if so.

Reply:
Nope. https://files.slack.com/files-pri/T0BD6KKA9-F836D6086/maintenance.png